libs/client/cornflow_client/core/experiment.py

```diff
@@ -168,9 +168,8 @@ def get_solver_config(
             if config.get("solver_config"):
                 conf.update(
                     {
-                        mapping[k, lib, solver]: v
+                        mapping.get((k, lib, solver), k): v
                         for k, v in config["solver_config"].items()
```

Comment:
the docstring of the method should also be edited to state that nothing is ever removed from solver_config